addon/services/flexberry-logging.js

```diff
@@ -39,21 +81,25 @@ export default Ember.Service.extend({
    */
   init() {
     this._super(...arguments);
-    this.set('flexberryStore', getOwner(this).lookup('service:store'));
-    this.set('serverLogEnabled', true);
+    this.set('_serverLogEnabled', true);
+    for (let enumName in this.logLevelEnums) {
+      let level = this.logLevelEnums[enumName];
+      this.enumsLoglevel[level] = enumName;
+    }
   },
 
   /**
-    Logger message to console and store.
-
-    @method flexberryLogger
-    @param {String} levelName Category name  - ERROR, WARN, LOG, INFO, DEBUG, DEPRECATION.
-    @param {String} message Message content.
-    @param {String} formattedMessage Full message content in JSON format.
-   */
-  flexberryLogger(levelName, message, formattedMessage) {
+  * Logger message to console and store
+  *
+  * @method flexberryLogger
+  * @param priority - priority
+  * @param levelName - category name  - ERROR, WARN, LOG, INFO, DEBUG, DEPRECATION
+  * @param message - message content
+  * @param formattedMessage - full message content in JSON format
+  */
+  flexberryLogger(priority, levelName, message, formattedMessage) {
     window.console.log(message);
```

Comment:
As i see task logging service must control the level logging message in the database, but does not control the level messages on the console. All messages as usual must displayed on console.